versions.yaml

```diff
@@ -14,24 +14,21 @@
 # with this program; if not, write to the Free Software Foundation, Inc.,
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
-
-_variables:
-  from9: &from9 gcr.io/google-appengine/debian9:latest
 versions:
```

Comment:
Keep 2.0 please. Redis-HA still uses it. But haproxy looks like good 👌🏻

Reply:
added to version.yaml 2.0 version